README.md

```diff
@@ -197,3 +197,7 @@ iOS13 and higher
 
 iOS12 and lower
 * NSBluetoothPeripheralUsageDescription
+
+#### How to add reactive ble lib to proguard rules (Android)
```

Comment:
How to make it clear _why_ users would want / need to adjust the ProGuard rules for our library? As I understand, it's not obvious that some problems are caused by this.

If nothing better, maybe the following version could be a bit better (otherwise capitalize abbreviations, don't use cut words [e.g. lib], stick to official spelling for product names [e.g. ProGuard]):

```
#### How to adjust ProGuard rules (Android)
```